Makefile

```diff
@@ -27,11 +30,7 @@ else
 	DOCKER_CONF := $(HOME)/.docker
 endif
 
-WORKDIR := $(shell mktemp -d)
-ifeq ($(shell uname),Darwin)
-	# Use a fixed folder to simplify limactl configuration (must be mounted with Write permissions)
-	WORKDIR := /tmp/serverless-workflows
-endif
+WORKDIR := ~/workdir
```

Comment:
are you sure this works when it's launched from a GH action, have you tested this use case? (e.g., "escalation" workflows are built this way)